scripts/anonymize.sql

```diff
@@ -186,8 +187,12 @@ UPDATE wiki_revisionip SET
 UPDATE wiki_revisionip SET
     referrer = CONCAT("https://example.com/", MD5(CONCAT(referrer, @common_hash_secret)))
     WHERE referrer != "";
+UPDATE wiki_revisionip SET data = null;
```

Comment:
Yes. ``data`` contains a JSON-serialized simple dictionary of request details, including sensitive information like IP and email address. ``RevisionIP`` objects are deleted after 30 days, this neuters the younger ones.